### PR TITLE
Support skipping missing values for certain transforms

### PR DESCRIPTION
**Pull request by ceferisbarov**
- Add `assert_continuous_or_missing` function which replaces `assert_continuous` in transforms that are going to support skipping missing values.
- Revise those transforms to support skipping missing values. Makes this an argument that defaults to true.
  
Opened this PR to give better idea about what I have in mind. Still working on tests.

### CODE REVIEW

File: src/assertions.jl [hunk 1]
```diff
@@ -7,3 +7,11 @@ function assert_continuous(table)
   types = schema(table).scitypes
   @assert all(T <: Continuous for T in types) "columns must hold continuous variables"
 end
+
+function assert_continuous_or_missing(table)
+  cols = Tables.columns(table)
+  for col in cols
+      type = scitype(col)
+      @assert type <: AbstractVector{<:Union{Missing, Continuous}} "columns must hold continuous variables"
```
Comment (juliohm) [inline src/assertions.jl:15]: Can you elaborate on why we are not using the same pattern of the assert_continuous function with T <: Union{<:Continuous,Missing} ?

**Review by juliohm — CHANGES_REQUESTED** [review]
This PR is doing two things at the same time, and I am still unsure about how we should handle missing values more generally. Missing values in Julia are still "work in progress" in the sense that people feel differently about the default model imposed by the language. Different ecosystems adopted different missing value models, and maybe we should delay this discussion to after the missing values model settles in the language and its data science community.

To give you some context, we are planning a v1.0 release this month, and it won't include support for missing values. Missing values could be considered in a future v2.0 release (after a year or so playing around with different models).

**Comment by juliohm** [comment]
Although we appreciate the effort, I think the best option we have right now is to close the PR and reconsider missing values in a year or so from now.

File: src/assertions.jl (hunk 1 above)
Comment (ceferisbarov) [reply src/assertions.jl:15]: I simply implemented the suggestion made in #27 .  I also agree that `T <: Union{<:Continuous,Missing}` is a more elegant solution.

**Pull request closed without merging**